sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeaders.java

```diff
@@ -15,6 +15,17 @@
  * A collection of headers on an HTTP request or response.
  */
 public class HttpHeaders implements Iterable<HttpHeader> {
+
+    /**
+     * The header which will have retry after value in milliseconds.
+     */
+    public static final String AZURE_RETRY_AFTER_MS_HEADER = "retry-after-ms";
+
+    /**
+     * The header which will have retry after value in milliseconds.
+     */
+    public static final String AZURE_X_MS_RETRY_AFTER_MS_HEADER = "x-ms-retry-after-ms";
```

Comment:
Add in Retry-After as an option, that is the standard HTTP header

Reply:
removing it based on previous comment from Jonathan.